app/src/main/res/layout/item_menu_drawer.xml

```diff
@@ -51,7 +54,22 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:gravity="center"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
             tools:text="42" />
 
-    </LinearLayout>
+        <ImageView
+            android:id="@+id/itemTick"
+            android:layout_width="wrap_content"
+            android:layout_height="16dp"
+            android:contentDescription="@string/contentDescriptionIconFolderSelected"
+            android:src="@drawable/ic_check"
+            android:visibility="gone"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:tint="?android:attr/colorPrimary" />
```

Comment:
Don't use a constraint layout, linear is fine enough. And use a FrameLayout to contain the bade and the checkmark if they can never be displayed together

Reply:
⚔️